modules/gitops-repo-server-properties.adoc

```diff
@@ -16,16 +16,18 @@ The following properties are available for configuring the repo server component
 |`serviceaccount` |`""` |The name of the `serviceaccount` to use with the repo-server pod.
 |`verifytls` |`false` |Defines whether to enforce strict TLS checking on all components when communicating with repo server.
 |`autotls` |`""` |Provider to use for setting up TLS for the repo-server's gRPC TLS certificate. Currently, only the `openshift` value is acceptable.
-|`image` | `argoproj/argocd` |The container image for Argo CD repo server. This overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.
-|`version` | same as `.spec.Version` |The tag to use with the Argo CD repo server.
-|`logLevel` | `info` |The log level used by the Argo CD repo server. Valid options are `debug`, `info`, `error`, and `warn`.
-|`logFormat` | `text` |The log format to be used by the Argo CD repo server. Valid options are `text` or `json`.
+|`image` | `argoproj/argocd` |The container image for Argo CD Repo server. This overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.
+|`version` | same as `.spec.Version` |The tag to use with the Argo CD Repo server.
+|`logLevel` | `info` |The log level used by the Argo CD Repo server. Valid options are `debug`, `info`, `error`, and `warn`.
+|`logFormat` | `text` |The log format to be used by the Argo CD repo server. Valid options are `text` and `json`.
 |`execTimeout` | `180` |Execution timeout in seconds for rendering tools (for example Helm or Kustomize).
 |`env` | `__<empty>__` |Environment to set for the repository server workloads.
```

Comment:
```suggestion
|`env` | `__<empty>__` |The environment to set for the repository server workloads.
```
Should "repository server" be "Repo server"?

Reply:
Yes, it should be "Repo server". Fixed.